File: exmms-core/src/main/java/gr/cite/exmms/core/DataElement.java
package gr.cite.exmms.core;

import java.util.ArrayList;
import java.util.List;

public class DataElement extends Element {
	
	private DataElement dataElement;

	private List<Collection> collections;
	
	public DataElement() {
		super();
		collections = new ArrayList<>();
	}
	
	public DataElement getDataElement() {
		return dataElement;
	}

	public void setDataElement(DataElement dataElement) {
		this.dataElement = dataElement;
	}

	public List<Collection> getCollections() {
		return collections;
	}

	public void setCollections(List<Collection> collections) {
		this.collections = collections;
	}
}

File: femme-web-ui/femme-admin/src/app/model/data-element.ts
import { SystemicMetadata } from './systemic-metadata';
import { Element } from './element';
import { Collection } from './collection';
import { Metadatum } from './metadatum';


export class DataElement extends Element {

	collections: Array<Collection>;
	dataElements: Array<Collection>;

	constructor() {
		super();
	}
}
